### ISSUE TITLE
Different forecasts give different elevations for the same point

### ISSUE THREAD

**Issue by psathyrella** [psathyrella/weatherscraper#5]
So forecasts from different sources can look substantially different for the same location, despite the underlying model being the same. This is frequently because they're giving you the forecast for different elevations. The real models, I think, are on a very course (~few km) scale, i.e. which doesn't include local topo/orographic whatnots. This gets added in afterward. And for the same reason that the coast of England is arguably infinite, the elevation at a given point is entirely dependent on the scale over which you average.

This is (I think) why [here](https://github.com/psathyrella/weatherscraper/issues/1) we were getting crazy different forecasts -- one person's topographic model had us roundabout mountaineer's creek while the other had us halfway up dragontail.

TLDR: elevations are approximate. Look at `mountain-forecast.com` if you want to be sure you're getting the elevation you expect.

More discussion [here](http://graphical.weather.gov/docs/ndfdSRS.htm)
